### PR TITLE
Fixes #1607 - Replace Latest Period of Daily Values with Instantaneous Values for Annual Flow Plot with Flood Frequency Statistics

### LINKED ISSUE #1607: Replace Latest Period of Daily Values with Instantaneous Values for Annual Flow Plot with Flood Frequency Statistics
See original issue #1424 and later issue #1537 for context. 

- [x] Replace the last 14 days (or some recent period) of daily values with instantaneous values (more frequent streamflow recordings). This will line up with the frequency of NWS Forecast data points and provide more context on the most recent streamflow. The call to NWIS for instantaneous values is: [https://nwis.waterservices.usgs.gov/nwis/iv/?format=json&sites=02173500&parameterCd=00060&startDT=1900-01-01](url)

### PR DESCRIPTION
**Pull request by maggiejaenicke**
Closes #1607

### CODE REVIEW

File: src/Controllers/GagePageController.ts [hunk 1]
```diff
@@ -2002,6 +2075,38 @@ module StreamStats.Controllers {
...
+            }
+            if (zoneAbbreviation === 'PST') {
+                timezone = 'America/Los_Angeles'
+            }
+            if (zoneAbbreviation === 'AKST' || zoneAbbreviation === 'AKDT') {
+                timezone = 'America/Anchorage' // test gage 15276000
+            }
+            if (zoneAbbreviation === 'HST') {
+                timezone === 'Pacific/Honolulu' // test gage 16759600
+            }
```
Comment (amedenblik) [inline src/Controllers/GagePageController.ts]: Do you know if this covers all the potential time zones?

**Review by amedenblik — COMMENTED** [review]
This looks great! I left one question/comment.

File: src/Controllers/GagePageController.ts (hunk 1 above)
Comment (maggiejaenicke) [reply src/Controllers/GagePageController.ts]: I made a few additions which I think should cover everything!

**Review by harper-wavra — APPROVED** [review]
Great job! This seems to be working well.

**Comment by amedenblik** [comment]
https://dev.streamstats.usgs.gov/gageplots/ has been updated

**Pull request merged**